react/components/MinicartIconButton.tsx

```diff
@@ -18,31 +18,42 @@ const countCartItems = (
   countMode: MinicartTotalItemsType,
   arr: OrderFormItem[]
 ) => {
-  if (countMode === 'distinctAvailable') {
-    return arr.reduce((itemQuantity: number, item: OrderFormItem) => {
+  if (countMode.indexOf('WithoutAssemblies') !== -1) {
```

Comment:
I think assemblies should never be counted. 

You can change the behavior of `distinctAvailable` and `totalAvailable`

Reply:
@klzns Its done ;) Anything wrong please contact me.